tools/process_raw_response.py

```diff
@@ -59,10 +58,7 @@ def camel_to_snake(name, upper=False):
 
     if args.provider == 'aws':
         object_format = 'raw_{}.get(\'{}\')'
-        cleaned_value = args.value.replace('<class \'dict\'>: ', '')
-        cleaned_value = args.value.replace('\}', '}')
-        cleaned_value = cleaned_value.replace(", tzinfo=tzlocal()", "")
-        cleaned_value = cleaned_value.replace(", tzinfo=tzutc()", "")
+        cleaned_value = args.value.replace('<class \'dict\'>: ', '').replace('\}', '}').replace(", tzinfo=tzlocal()", "").replace(", tzinfo=tzutc()", "")
```

Comment:
Better keep these separate, it's not clean and we should really use regexes, but this way seems worst.

Reply:
Will revert per-your preference, IMHO they are equally legible and this way is "technically correct" though